tests/nnstreamer_sink/unittest_sink.cc

```diff
@@ -2938,6 +2946,85 @@ TEST (tensor_stream_test, octet_2f)
   _free_test_data ();
 }
 
+/**
+ * @brief Test for octet stream.
+ */
+TEST (tensor_stream_test, octet_multi_tensors)
```

Comment:
Need more negative cases for unittest_sink.cc (2 more).

Reply:
add 7 negative tc in unittest_plugins